libbeat/_meta/fields.common.yml

```diff
@@ -59,3 +59,47 @@
           type: keyword
           description: >
             Error type.
+
+- key: alias
+  title: Alias
+  description: >
+    Alias fields for compatibility with 7.x.
+  fields:
+    # Common Beats fields
+    - name: agent.type
+      type: alias
+      path: beat.name
+    - name: host.hostname
```

Comment:
@webmat Is that what you expect here?

Reply:
I'm not as familiar with the semantics of the old fields.

- `beat.hostname` ~ `host.hostname` makes sense to me
- Was the `beat.name` field meant to be customizable by users, like the pattern we've established in ECS? If so, perhaps we should make it equivalent to `agent.name`? Right now if I look at Metricbeat and Packetbeat data, `beat.name` is the hostname of my laptop, it's not an [agent type](https://github.com/elastic/ecs#user-content-agent.type) like `filebeat` or `packetbeat`.